src/ui/component/selectThumbnail/view.jsx

```diff
@@ -64,7 +63,7 @@ class SelectThumbnail extends React.PureComponent<Props, State> {
 
     const { thumbnailError } = this.state;
 
-    const isSupportedVideo = getMediaType(null, filePath) === 'video';
+    const isSupportedVideo = Lbry.getMediaType(null, filePath) === 'video';
```

Comment:
out of scope for this PR, but I wonder if putting all of these on the `Lbry` object is the right design

Reply:
We also have the file name with extension in the claim data now, but we should fall back to file list when it's not available

Reply:
Since that only has _some_ of the media types, I don't think it's worth it to use for now, especially since it's working pretty well.

And yeah, this is probably better as a regular utility function?